Noemi_first_try/index/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ich hätt hier mal paar Fragen</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+  
+  <header><h1> Ich hätt hier mal ne...</h1></header>
+  <home><a href="../index/index.html">Home</a></home>
+      <p> Es gibt unzählig viele Fragen auf die es kein Richtig oder Falsch gibt. Solche Fragen sind unheimlich spannend. Mit anderen diskutiert, erhält man immer wieder einen neuen Blickwinkel. Sei das mit Freunden, Grosseltern, Familie oder Leuten, die man eben erst kennengelernt hat.</p>
+      <p> Die Fragen hier dienen der Inspiration solltet ihr mal mit ner fremden Person im Lift stecken bleiben, oder ihr mal Lust habt eure Freunde und Familie  neu kennenzulernen. Ich wünsche viel Spass bei gesellschaftlichen Runden mit Freunden und Familie und unzählige neue Erkenntnisse!</p>
+      
+    <h2>... Frage zu nem Thema.</h2>
+    <p>
+      <div class="box1"><a href="../questions/talent.html"><h3>Talent</h3></a></div>
+      <div class="box2"><a href="../questions/meinung.html"><h3>Meinung</h3></a></div>
+      <div class="box3"><a href="../questions/angst.html"><h3>Angst</h3></a></div>
+      <div class="box4"><a href="../questions/erwartungen.html"><h3>Erwartungen</h3></a></div>
+      <div class="box5"><a href="../questions/kinder.html"><h3>Kinder</h3></a></div>
+      <div class="box6"><a href="../questions/essen.html"><h3>Essen</h3></a></div>
+      <div class="box7"><a href="../questions/hoffnung.html"><h3>Hoffnung</h3></a></div>
+      <div class="box8"><a href="../questions/geschichte.html"><h3>Geschichte</h3></a></div>
+    </p>
```

Comment:
Nice that you use boxes with links for each topic and nice layout! 👍 

Here I would suggest on line 18 to use `<div>`  instead of `<p>` to use as a "container" for the other boxes, and call it something like `<div class="box-container">`. Then in CSS you could either use Flexbox or CSS Grid, for example by using flex-direction: row and setting the width of each box to below 25 %.

Some suggestions are:
Flexbox
https://css-tricks.com/snippets/css/a-guide-to-flexbox/
CSS Grid
https://css-tricks.com/snippets/css/complete-guide-grid/

Another suggestion is to name the boxes something like "box-one", "box-two" etc. using kebab-case for CSS elements.